run_scripts/run_pipeline.sh

```diff
@@ -14,21 +14,22 @@ fi
 USER=$1
 GOOGLE_CLOUD_CREDENTIALS_FILE_PATH=$2
 PIPELINE_CONFIGURATION=$3
-CODA_PULL_CREDENTIALS_PATH=$4
-CODA_PUSH_CREDENTIALS_PATH=$5
-CODA_TOOLS_ROOT=$6
-DATA_ROOT=$7
-DATA_BACKUPS_DIR=$8
-PERFORMANCE_LOGS_DIR=$9
+PIPELINE_RUN_MODE=$4
+CODA_PULL_CREDENTIALS_PATH=$5
+CODA_PUSH_CREDENTIALS_PATH=$6
+CODA_TOOLS_ROOT=$7
+DATA_ROOT=$8
+DATA_BACKUPS_DIR=$9
+PERFORMANCE_LOGS_DIR=${10}
 
 RUN_ID=$(date -u +"%Y-%m-%dT%H:%M:%SZ")
 
-./1_coda_get.sh "$CODA_PULL_CREDENTIALS_PATH" "$CODA_TOOLS_ROOT" "$DATA_ROOT"
+#./1_coda_get.sh "$CODA_PULL_CREDENTIALS_PATH" "$CODA_TOOLS_ROOT" "$DATA_ROOT"
```

Comment:
I don't think you meant to commit this commented

Reply:
Yes yes, thanks for catching this.